src/components/Accordion/Accordion.tsx

```diff
@@ -54,7 +56,7 @@ const Accordion = ({ headingLevel = 'h2', isContained = false, children }: Accor
   };
 
   return (
-    <div className={'bsds-accordion' + (isContained ? ' is-contained' : '')}>
+    <div className={useConcatenation(['bsds-accordion', `${isContained ? 'is-contained' : ''}`, `${className || ''}`])}>
```

Comment:
let's chat about your thoughts for the hook vs checking a class is provided, e.g.:

```<div className={'bsds-accordion' + (isContained ? ' is-contained' : '') + (className !== '' ? ` ${className}` : '')}>```